backend/src/main/java/de/qaware/mercury/business/image/ImageService.java

```diff
@@ -12,6 +12,15 @@
 
     InputStream loadImage(Image.Id imageId) throws ImageNotFoundException;
 
+    /**
+     * Get the probable background color of the image with the given ID.
+     *
+     * @param imageId the internal ID of the image as referenced in the shop.
+     * @return A hex color string (e.g. '#FFAF10', including the '#' sign).
+     * @throws ImageNotFoundException If the image with the given ID cannot be found.
+     */
+    String getImageBackgroundColor(Image.Id imageId) throws ImageNotFoundException;
```

Comment:
Wieso gibt das String und nicht `Color` zurück?

Reply:
Weil Color die interne Repräsentation in dem Modul ist. Der Datentyp prüft nicht, ob Hex und RGB zusammenpassen o.ä. und deshalb wollte ich den nicht nach außen reichen. Alles was du außen noch brauchst, ist die Hex Farbe.

Reply:
Na gut :)